pkg/bpf/bpf.go

```diff
@@ -89,6 +89,10 @@ func Start() error {
 		return err
 	}
 
+	if err = mountCgroup2(&config); err != nil {
+		return err
```

Comment:
AFAIK, cgroup v2 only available in linux kernel v5.8 or above. 

Is mouting cgroup v2 a necessary step use kmesh? If not we could just print a warning message and not return error directly, otherwise kmesh could not run on older versions of the kernel. 

I guess v5.8 is a pretty high version for production environments.

Reply:
https://github.com/kmesh-net/kmesh/blob/main/build/docker/README.md#start_kmeshsh

It is stated that to start kmesh, have to mount cgroupv2.